packages/lit-protocol-cipher/test/index.test.ts

```diff
@@ -88,6 +99,55 @@ describe('LitProvider', () => {
     });
   });
 
+  describe('encryption and decryption state', () => {
+    it('should manage decryption state correctly', () => {
+      expect(litProvider.isDecryptionEnabled()).toBe(false);
+
+      litProvider.enableDecryption(true);
+      expect(litProvider.isDecryptionEnabled()).toBe(true);
+
+      litProvider.enableDecryption(false);
+      expect(litProvider.isDecryptionEnabled()).toBe(false);
+    });
+
+    it('should check encryption availability', () => {
+      // First verify it's true with a valid client
+      expect(litProvider.isEncryptionAvailable()).toBe(true);
+
+      // Mock the implementation to simulate no client
+      jest.spyOn(litProvider, 'isEncryptionAvailable').mockImplementation(() => false);
+
+      // Now it should return false
+      expect(litProvider.isEncryptionAvailable()).toBe(false);
```

Comment:
This is just exercising the mock lib rather than the actual code. Not sure the best solution but consider mocking the litClient to undefined.

Reply:
Yes, good catch. changing it